persistence/persistence-jdbc/src/main/java/uk/gov/justice/services/jdbc/persistence/JdbcRepositoryHelper.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.justice.services.jdbc.persistence;
+
+import static uk.gov.justice.services.jdbc.persistence.PreparedStatementWrapper.valueOf;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Spliterator;
+import java.util.Spliterators;
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+import javax.sql.DataSource;
+
+public class JdbcRepositoryHelper<T> {
```

Comment:
Maybe have a javadoc comment explaining what this is and the Type.

Reply:
done